README.md

```
@@ -155,6 +158,25 @@ pnpm migrate:prod
 pnpm seed
 ```
 
+> **Never run `pnpm prisma migrate deploy` directly** — it picks up `.env.local` (the dev DB).
+> Always use `pnpm migrate:prod`, which explicitly loads `.env` (the production DB) and skips `.env.local`.
+
+#### Adding a new model — workflow
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)